skt/executable.py

```diff
@@ -310,7 +310,8 @@ def cmd_build(args):
     # Get the SHA of the commit from the repo that we just compiled.
     buildhead = get_state(args['rc'], 'buildhead')
 
-    if tgz:
+    # Handle any built tarballs.
+    if package_path and package_path.endswith('tar.gz'):
```

Comment:
I wonder what happens if package_path is None :stuck_out_tongue:

Reply:
The tarball + rpm_repo don't get written to the state file in that case.

Reply:
OK, but do we want to keep going if that happens?